### ISSUE TITLE
SIT Wording about Definition of "Quotation", 2 Verses

### ISSUE THREAD

**Issue by ARMediting** [gryphonshafer/Quizzing-Rule-Book#119]
The final bullet point under "Quotation" in 1.2.2.5. (SIT) says that "the beginning of the SIT question must start within the 1st 2 verses of the quotation." Combined with the earlier bullet point that "the quotation must not exceed 2 consecutive verses," this seems to allow for a SIT quotation coming from the 2nd and 3rd verses of a quote in Scripture. Is this what's intended? I've never seen a SIT written that way before.

This is muddied by the word "quotation" being used to refer to both the entire amount of a statement made by a person in the narrative (regardless of length) and the text of a SIT. The two should be defined separately, perhaps by "Statement" and "Quotation."

**Comment by scottpeterson** [comment]
Yes, a valid SIT can come from verses 2 and 3 of a quotation that begins in verse 1. Now, there's rarely a good reason to do so, but it's valid. 

Yep, I'd also like to differentiate between those two uses of the word "quotation."

**Comment by jswingle** [comment]
Hard to know if this is what's "intended" but it's certainly how it reads, and I've written plenty of questions this way in my own sets.

**Comment by ARMediting** [comment]
I like your ideas. Do you think it's necessary to include something about ending at the end of a sentence? What if the 2nd verse of the quotation isn't the end of a sentence or a thought?

- "Must end at the end of a sentence or the end of dialog." [I'm trying to say if the quote ends and then the sentence ends with "he said" or something, that counts too. What can we call that?]

Also, in the paragraph below the bullet point, should we clarify that that 2 separate quotations being combined are said by the same person?

**Comment by josiah-leinbach** [comment]
I agree those are relevant considerations, but I think they are best left to the question writing and editing process rather than codified in the rulebook. For what is is worth, in sixteen years of quizzing I have never come across a SIT question that (1) cuts off mid sentence or (2) combines quotes by two different speakers, largely because the writing/editing process is pretty good on this stuff.

**Comment by ARMediting** [comment]
Should we take into consideration a quizzer evaluating potential questions during their own study time? As a former quizzer who's studied for Internationals, I always wanted to know what I should include or exclude on a list, and not seeing a specific variation on a question type during the year is no indication of whether it might pop up at IBQ.

**Comment by scottpeterson** [comment]
Remember Alex "sentence" is VERY VERY hard to define. It's defined not by punctuation, but by structure of thought. (my research and knowledge may be out of date--please correct me if so).

**Comment by jswingle** [comment]
Just to progress the discussion a little more: while I wouldn't be dead-set opposed to changing the rules so that the quotation can't go past the second verse from the start of the quotation, (even if the SIT question starts in the 2nd verse), I don't think there's much reason to change it either. I also think the rulebook as written is pretty clear and unambiguous, even though a lot of people may not write questions that way or be aware of it. So I'm not in favor of a change here.

Although: I sort of have the opinion that we should get rid of this rule altogether, as well as all context rules when it comes to SITs. For example, I think it would be asking something meaningful to require quizzers to identify "Jesus said it on a mountain" for quotations from the Sermon on the Mount that aren't at the beginning of Matthew 5. I feel like this is such a large-scale change that it belongs in a separate issue, however. (And I don't particularly feel like opening said issue right now, we have more important fish to fry)

**Comment by ARMediting** [comment]
Here's an idea for revised text with: "Quotation" only meaning the part of the quote on the paper for the quizzer to give, and other phrasing for all spoken text by a person in the passage; added clarification that the quotation always ends before the third verse; rearranged the bullet points in order from least specific to most; cleaned up bottom paragraph with some clarifications and rearrangement of what's there. (Changes in bold)

> - Quotation
> - Must be direct **spoken text** from the material by a person or an entity
> - Must start at the beginning of **spoken dialogue as indicated by quote marks** or the beginning of a sentence within the dialogue
> - When a SIT question comes from **speech** that runs for more than 2 verses, the beginning of the **quotation** must start within the first 2 verses of the **dialogue**
> - Must not exceed 2 consecutive verses **or extend beyond the second verse of a longer section of uninterrupted speech**
> - The first 5 words must be unique to all quotations
> - The quizzer is not required to repeat the quotation word-perfect but must be substantially close without leaving out key components of the quotation
> 
> If the quotation is split-up by non-quotation words **(outside of quote marks)**, the quotation will be **written and** read without the non-quotation words. A SIT question can combine 2 separate quotations **by the same person** as long as no words are added and the 2 quotes flow easily from one to another.

**Comment by kclimenhaga** [comment]
In general, I agree, but I think you have raised a few points that need to be addressed:

1. Phrasing it as "spoken text" eliminates all SITs that are from written text (mainly quotations from the Old Testament). I know at the district level, we desperately need those SITs for our first meet of the year, because the start of most gospels has a lot more quotations from Scripture than narrative with dialogue.

2. Limiting it to the first two verses of what I'll call the utterance (not sure what else to call it if we don't want to use "quotation") would exclude SITs where verse 1 is mostly speaker tag with a tiny bit of speaking, and then the logical situation is from verses 2-3. The best example I can think of is Acts 4:8-10:
8	Then Peter, filled with the Holy Spirit, said to them: “Rulers and elders of the people!
9	If we are being called to account today for an act of kindness shown to a man who was lame and are being asked how he was healed,
10	then know this, you and all the people of Israel: It is by the name of Jesus Christ of Nazareth, whom you crucified but whom God raised from the dead, that this man stands before you healed.

You could make a SIT just on verse 8, but you can't make it on verses 8-9. To me, the logical SIT would be verses 9-10.

If we want to exclude these kinds of SITs (there aren't many), that's fine, but I want to make sure this is considered before the wording is changed.

As a completely unrelated point, I'd like to consider rewording "The first 5 words must be unique to all quotations" to "The first 5 words must be unique within the material" or something like that. If the first 5 words occur somewhere else that happens not to be in a quotation, I don't think we should use that question.

**Comment by lukeshev** [comment]
I agree with Kristen.
